doc/api/index.rst

```diff
@@ -74,6 +74,7 @@ Operations on grids:
     grdcut
     grdfilter
     grdtrack
+    grd2cpt
```

Comment:
Maybe shift this to be under the `Color palette table generation:` section (above `makecpt`).

Reply:
Moved it in 1812b93